test/functional/feature_remove_pruned_files_on_startup.py

```diff
@@ -50,6 +51,20 @@ def run_test(self):
         assert not blk0.exists()
         assert not rev1.exists()
 
+        self.log.info("Check that a reindex will wipe all files")
+
+        def file_count():
+            return sum(
+                entry.is_file() and any(map(entry.name.startswith, ["rev", "blk"]))
+                for entry in self.nodes[0].blocks_path.iterdir()
+            )
+
+        assert_equal(file_count(), 4)
```

Comment:
nit:
If retouching, could increase readability with a comment or log explaining that the check is for presence of non-00000/00001 files (blk00002.dat/rev00002.dat/blk00003.dat/rev00003.dat)